packages/fulfillment/src/services/service-provider.ts

```diff
@@ -0,0 +1,103 @@
+import {
```

Comment:
question: Should we call this `FulfillmentProvider` as in V1, or are there some specific thoughts behind the choice of name here? Think it gets a bit messy with `ServiceProviderService`.

Reply:
Agreed with Oli here